bin/rebuild

```diff
@@ -31,6 +32,38 @@ versiondb_head() {
   (cd "$VERSIONDB" && git rev-parse HEAD)
 }
 
+save_build_env() {
+  # saving environment information
+  filename="${LSSTSW_BUILD_DIR}/${BUILD}.env"
+  echo "Saving environment information in ${filename}"
+  conda env export | grep -v "^name"> /tmp/lenv
+  echo "environment_name: ${LSST_CONDA_ENV_NAME}" > "${filename}"
+  cat /tmp/lenv | grep "^prefix" >> "${filename}"
+  cat /tmp/lenv | grep -v "^prefix" > /tmp/lenv1
+  if [[ "${LSST_CONDA_ENV_NAME}" != "lsst-scipipe" ]]; then
+    ref="${LSST_CONDA_ENV_NAME:12}"
+    if [[ $ref == *"."* ]]; then
+      ref1=`echo "${ref}" | cut -f 1 -d .`
+      ref2=`echo "${ref}" | cut -f 2 -d .`
+      local_env_dir="${LSSTSW}/env/${ref1}/${ref2}"
+    else
+      local_env_dir="${LSSTSW}/env/${ref}"
+    fi 
+    if [ -d "${local_env_dir}" ]; then
+      ref_file=`ls "${local_env_dir}"`
+      local_env_file="${local_env_dir}/${ref_file}"
+      echo "env_ref_file: ${local_env_file}" >> "${filename}"
+      cat "${local_env_file}" | grep -v "^name" > /tmp/renv
+      set +e  # disable set -e for the diff commands.
+      sdiff /tmp/lenv1 /tmp/renv >> "${filename}"
```

Comment:
An `sdiff` seems hard to use by any other automation.  Why not use the `diff -u` here, if anything is needed at all?